### ISSUE TITLE
Possibility to turn off statistics

### ISSUE THREAD

**Issue by VanDavv** [denilsonsa/prettyping#18]
There is no possibility to turn off statistics.

It would be nice to reproduce the same functionality as for `--nolegend`, but with `--nostatistics`

**Comment by oscarfroberg** [comment]
Yes please, and/or to limit statistics to `n` or if not specified, to whatever `--last` is set to?
